registry_module.go

```diff
@@ -33,6 +33,9 @@ type RegistryModules interface {
 	// Create and publish a registry module with a VCS repo
 	CreateWithVCSConnection(ctx context.Context, options RegistryModuleCreateWithVCSConnectionOptions) (*RegistryModule, error)
 
+	// Create and publish a registry module with a VCS repo
+	CreateWithGithubApp(ctx context.Context, organization string, options RegistryModuleCreateWithVCSConnectionOptions) (*RegistryModule, error)
```

Comment:
I feel like this distinction isn't so important because the options are identical, with the only difference being that an org name is required for the URL. I wonder if it would be better to include organization in the RegistryModuleCreateWithVCSConnectionOptions, but only require it if GHAInstallationID is also present. Then you'd choose your URL based on which options are present, potentially preferring one over the other.

OR if you think there is a relevant distinction between VCS and GithubApp in the mind of users, then emphasize that by splitting the options into VCS options and GIthubApp options.

What do you think?